toolkits/springernature/packages/springernature-header/scss/10-settings/fonts.scss

```diff
@@ -0,0 +1 @@
+$fontMerriweather: 'Merriweather', serif;
```

Comment:
This also [comes with the brand-context](https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/springernature/scss/10-settings/typography.scss#L6)